x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/discover/cell_actions.cy.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { grantClipboardReadPerm } from '../../../../tasks/common/clipboard';
+import {
+  DISCOVER_CELL_ACTIONS,
+  DISCOVER_CONTAINER,
+  DISCOVER_FILTER_BADGES,
+  GET_DISCOVER_DATA_GRID_CELL,
+} from '../../../../screens/discover';
+import { waitForDiscoverGridToLoad } from '../../../../tasks/discover';
+import { updateDateRangeInLocalDatePickers } from '../../../../tasks/date_picker';
+import { login, visit } from '../../../../tasks/login';
+import { createNewTimeline, gotToDiscoverTab } from '../../../../tasks/timeline';
+import { ALERTS_URL } from '../../../../urls/navigation';
+
+const INITIAL_START_DATE = 'Jan 18, 2021 @ 20:33:29.186';
+const INITIAL_END_DATE = 'Jan 19, 2024 @ 20:33:29.186';
+const TIMESTAMP_COLUMN_NAME = '@timestamp';
+
+describe(
+  `Discover Datagrid Cell Actions`,
+  { env: { ftrConfig: { enableExperimental: ['discoverInTimeline'] } } },
```

Comment:
Do you also need to add the serverless tag? `{ tags: [tag.SERVERLESS] }`

Reply:
Actually, all the tests are failing in serverless because of full screen issue and at the time, serverless PR was not merged yet. Currently, tests run in ESS mode. I will enable them in serverless in follow-up PR.